hr_attendance_timesheet_adj/views/hr_timesheet_sheet_sheet_views.xml

```diff
@@ -21,6 +21,24 @@
                     </div>
                 </button>
             </xpath>
+            <xpath expr="//page[3]" position="replace">
+            </xpath>
```

Comment:
Can you please check if making the page invisible would also work (i.e. present all the attendance records correctly in the first page). If that is the case, we would like to go for that option in preference of not using replace.  If invisible attribute does not work then we will go with replace.

Reply:
Hi @yostashiro It'seems not working with invisible.
![Peek 2020-04-13 12-18](https://user-images.githubusercontent.com/24264979/79099144-2009fc80-7d81-11ea-8461-6e85474e0383.gif)